### ISSUE TITLE
Keeps in "loading" when typing the keyword for device switch

### ISSUE THREAD

**Issue by SonOfSkywalker** [CeroProgramming/ulauncher-pulsecontrol#9]
The volume keyword is working fine but when I want to use the Device Switcher command, it gets stuck in "loading" mode. PulseAudio is installed and I can see my devices in the application. Thanks for your help.

**Comment by YetAnotherSimon** [comment]
Hi. Sorry I haven't answered before, I was quite busy the last months. If you still have the issue, can you tell me a bit about your system (os etc.)? And maybe launch ulauncher from the cli and send the output of the application when you try to replicate the issue. Make sure to close ulauncher fully before starting from the cli. If you need help feel free to reach out.

**Comment by azulli** [comment]
Hi, I'm also using this plugin and I found out that the pactl command prior to version 16.0 formats the numbers using the  LC_NUMERIC configured setting, `it_IT.UTF-8` in my specific case.

Prefixing the call with `LC_NUMERIC=C` returns the correct values, fixing the issue.

**Comment by YetAnotherSimon** [comment]
Interesting and unexpected. Anyway I created a new branch [lc_locale_fix](https://github.com/YetAnotherSimon/ulauncher-pulsecontrol/tree/lc_locale_fix) and added a very quick and honestly untested fix because I am a bit in a hurry and have no system with the right pactl at hand anyway. If someone wants to test it I'd appreciate it otherwise I will try to find some time but I cannot say when I can do this

**Comment by azulli** [comment]
Hi, just checked and the fix works well in my installation with pulseaudio-utils 15.99.1.
Thanks a lot.

**Comment by YetAnotherSimon** [comment]
Great, merged it! I will close the issue, if any problems remain, feel free to open a new issue or reopen this issue.
